doc/developer_guide/index.md

```diff
@@ -51,11 +55,34 @@ The `.pixi` directory can be large, so it is advised not to put the source direc
 
 The source code for the Panel project is hosted on [GitHub](https://github.com/holoviz/panel). The first thing you need to do is clone the repository.
 
+::::{tab-set}
+
+:::{tab-item} Contributor
+
 1. Go to [github.com/holoviz/panel](https://github.com/holoviz/panel)
 2. [Fork the repository](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#forking-a-repository)
 3. Run in your terminal: `git clone https://github.com/<Your Username Here>/panel`
 
+:::
+
+:::{tab-item} Core Contributor
```

Comment:
Agree.

Reply:
I agree that you know how to do it.

But as any contributor you go here to get installed. Its multiple steps and they change over time.

It is convenient you can Copy into terminal instead of spdnding additional time compsing commands.